src/multidim/intervalbox.jl

```diff
@@ -45,7 +45,7 @@ Return a vector of the `mid` of each interval composing the `IntervalBox`.
 See `mid(X::Interval, α=0.5)` for more informations.
 """
 mid(X::IntervalBox) = mid.(X)
-mid(X::IntervalBox, α) = mid.(X, α)
+mid(X::IntervalBox, α) = mid.(X[:], α)
```

Comment:
This can be left as `mid.(X, alpha)` after incorporating the change in #159

Reply:
Shall I wait that #159 to change it, or should I leave it like this for the time being? (#161 has some difficulties related to this.)